examples/argocd-with-applications/main.tf

```diff
@@ -125,9 +125,17 @@ module "ingress" {
   tags = local.tags
 }
 
-module "monitoring" {
+# module "monitoring" {
+#   module_depends_on = [module.argocd.state.path]
+#   source            = "../../modules/monitoring/prometheus"
+#   cluster_name      = module.kubernetes.cluster_name
+#   argocd            = module.argocd.state
+#   domains           = local.domain
+# }
+
+module "victoriametrics_monitoring" {
   module_depends_on = [module.argocd.state.path]
-  source            = "../../modules/monitoring/prometheus"
+  source            = "../../modules/monitoring/victoria-metrics"
   cluster_name      = module.kubernetes.cluster_name
   argocd            = module.argocd.state
```

Comment:
In the example or readme to the module, it would be great to have a description of critical parameters such as vmselect. PersistentVolume.size or vmselect.StatefulSet.enabled so that you don't have to study the charts every time before deploying.

Reply:
I fixed resource limits.